Exiled.Events/EventArgs/Player/AimingDownSightEventArgs.cs

```diff
@@ -23,19 +24,18 @@ public class AimingDownSightEventArgs : IPlayerEvent, IFirearmEvent
         /// <param name="player">
         ///     <inheritdoc cref="Player" />
         /// </param>
+        /// <param name="firearm">
+        ///     <inheritdoc cref="Firearm" />
+        /// </param>
         /// <param name="adsIn">
         ///     <inheritdoc cref="AdsIn" />
         /// </param>
         /// <param name="adsOut">
         ///     <inheritdoc cref="AdsOut" />
         /// </param>
-        public AimingDownSightEventArgs(Player player, bool adsIn, bool adsOut)
+        public AimingDownSightEventArgs(Player player, Firearm firearm, bool adsIn, bool adsOut)
```

Comment:
That's a breaking change, we're changing the signature of a `public` constructor.